src/ExpressionParser.php

```diff
@@ -774,16 +774,13 @@ private function getTestNodeClass(TwigTest $test): string
             $stream = $this->parser->getStream();
             $message = \sprintf('Twig Test "%s" is deprecated', $test->getName());
 
-            if ($test->getDeprecatedVersion()) {
-                $message .= \sprintf(' since version %s', $test->getDeprecatedVersion());
-            }
             if ($test->getAlternative()) {
                 $message .= \sprintf('. Use "%s" instead', $test->getAlternative());
             }
             $src = $stream->getSourceContext();
             $message .= \sprintf(' in %s at line %d.', $src->getPath() ?: $src->getName(), $stream->getCurrent()->getLine());
 
-            @trigger_error($message, \E_USER_DEPRECATED);
+            trigger_deprecation('twig/twig', $test->getDeprecatedVersion() ? $test->getDeprecatedVersion() : '', $message);
```

Comment:
what if the test does not come from the `twig/twig` package ?

Reply:
I don't have a good answer unfortunately as this can come from both internally and from a third-party.
We could potentially make it configurable, but I'm not sure this is worth it.
Can we use just 'Twig' as the "package" name? Or maybe an empty package name?

Reply:
An empty package name is supported, so that's maybe the best option.

Reply:
We should probably add support for providing a package name alongside the version.